README.md

```diff
@@ -281,7 +281,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [IP Geolocation](https://ipgeolocation.io/) — IP Geolocation API - Forever free plan for developers with 30k requests per month (1k/day) limit.
   * [IP Geolocation API](https://www.abstractapi.com/ip-geolocation-api) — IP Geolocation API from Abstract - Extensive free plan allowing 20,000 monthly requests.
   * [IP2Location](https://www.ip2location.com) — Freemium IP geolocation service. LITE database is available for free download. Import the database in the server and perform a local query to determine the city, coordinates, and ISP information.
-  * [IP2Location.io](https://www.ip2location.io/) — Freemium, fast and reliable IP geolocation API to determine geolocation data like city, coordinates, ISP, etc. The free plan is available with 30k credits per month. Subscribe to paid plans for more advanced features or contact us for a personalized plan.
+  * [IP2Location.io](https://www.ip2location.io/) — Freemium, fast and reliable IP geolocation API to determine geolocation data like city, coordinates, ISP, etc. The free plan is available with 30k credits per month. IP2Location.io also offers WHOIS lookups and hosted domain lookup services for free, lets you access domain registration details like ownership, registrar, and key dates such as creation and expiration. The Hosted domain lookup reveals which domains are hosted on a specific IP address, helping identify shared hosting environments. Subscribe to paid plans for more advanced features or contact us for a personalized plan.
```

Comment:
This is a quite long addition for quite common services. Please shorten. The requested change was for a short addition 

This list is very big it’s in your interest to keep these entries short and easy to consume,

thank you!